### ISSUE TITLE
Remove Relationship Status from edit profile.

### ISSUE THREAD

**Issue by bsteinbeigle** [Methodician/guytime#97]

**Comment by bsteinbeigle** [comment]
@kaxline relationship status should be remove from edit profile. 
![Relationship Status Change](https://user-images.githubusercontent.com/89544670/135920011-7b50f589-bc72-44f1-bc27-94a7675d4746.jpg)

**Comment by bsteinbeigle** [comment]
@kaxline What's your thoughts on adding locations? I was thinking we could include towns/cities near Salt Lake City, Utah. I was thinking a drop down list may be the most economical way to accomplish this at this stage in development. Jacob mentioned that geofencing is one way to accomplish this but I believe that will take too much of a development effort at this juncture. I can provide you with a list of 10-15 city/town names out there if that is helpful? On the profile I was thinking we could display the persons location directly below the guys name and below that have age. Thoughts? If this a large development effort I would rather focus our time on adding the new events functionality I will be sending over shortly.

**Comment by kaxline** [comment]
@bsteinbeigle Yeah, that sounds good. Want to create a new issue with the list of city names? It's not too heavy a lift and can be rolled into the changes I already need to make to that screen.

**Comment by bsteinbeigle** [comment]
@kaxline Will do. Be on the lookout for the new issue. Guess we are veering off this issues topic. I think I will be getting my town city list from the public transit train. They did a great job cutting out all the small little towns/cities out here;)
